### DISCUSSION TITLE
Creating files

### DISCUSSION THREAD

**Discussion by badson18** [thebaselab/codeapp#988] [Ideas]
I would love if i could create files in subfolders, instead of main folder i have opened. It would make creating new subprograms and files much easier.

**Comment by bummoblizard** [comment]
Thanks for opening the discussion. You can actually do this by holding the folder cell and select 'create file'.

**Comment by badson18** [comment]
i didn’t saw that thank you!

W dniu pon., 18.12.2023 o 09:41 Chung Shing Hin ***@***.***>
napisał(a):
